WcaOnRails/db/migrate/20170523034604_create_post_tags.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreatePostTags < ActiveRecord::Migration[5.0]
+  def change
+    create_table :post_tags do |t|
```

Comment:
Haha, yeah. There's a chance they changed something in Rails 5 and this "just works", so it's good we checked =)